pkg/neorpc/result/version.go

```diff
@@ -14,11 +14,13 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		TCPPort   uint16   `json:"tcpport"`
-		WSPort    uint16   `json:"wsport,omitempty"`
-		Nonce     uint32   `json:"nonce"`
-		UserAgent string   `json:"useragent"`
-		Protocol  Protocol `json:"protocol"`
+		TCPPort                uint16   `json:"tcpport"`
+		WSPort                 uint16   `json:"wsport,omitempty"`
+		Nonce                  uint32   `json:"nonce"`
+		UserAgent              string   `json:"useragent"`
+		Protocol               Protocol `json:"protocol"`
+		MaxIteratorResultItems int      `json:"maxiteratorresultitems"`
+		SessionEnabled         bool     `json:"sessionenabled"`
```

Comment:
You haven't ported the https://github.com/neo-project/neo-modules/pull/878. It should be a separate `rpc` section:
https://github.com/neo-project/neo-modules/blob/12ccd6810248a7d89670a9a8f72c46a33cb39810/src/RpcServer/RpcServer.Node.cs#L98